pynear/__init__.py

```diff
@@ -1,4 +1,4 @@
-# flake8: noqa
+from _pynear import BKTreeBinaryIndex64
```

Comment:
I can see we use now 64 bit keys only. So should we add some check in python (like I did with vpbinary index) to make sure input has dimension 8 and type uint8? The hamming64 distance assumes this.

Reply:
Also, the most tipical binary descriptor dimension is 256. Maybe we can also add 128, 256 descriptors (we already have optimized hamming distances for those dimensions).

Reply:
yes the PR is not complete yet, I am planning to add the other vector sizes. But I would also like to add generic hamming distances (maybe optimized for multiples of 8, 16, 32 or so, but otherwise independent of size) like our L2 metrics.
Also I want to refactor the bindings for these cases a bit to avoid code duplication. Right now it's a bit messy with lots of copy-pasted code.

Reply:
Totally agreed. This bindings are also making me nervous with so much repeated code. I was also planning to create a base class. Thanks for handling this!